File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/com/github/asyncmc/mojang/authentication/scala/gatling/model/RefreshResponse.scala

package com.github.asyncmc.mojang.authentication.scala.gatling.model


case class RefreshResponse (
    _accessToken: String,
    _clientToken: Option[String]
)
object RefreshResponse {
    def toStringBody(var_accessToken: Object, var_clientToken: Object) =
        s"""
        | {
        | "accessToken":$var_accessToken,"clientToken":$var_clientToken
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/com/github/asyncmc/mojang/authentication/kotlin/spring/model/RefreshResponse.kt
package com.github.asyncmc.mojang.authentication.kotlin.spring.model

import java.util.Objects
import com.github.asyncmc.mojang.authentication.kotlin.spring.model.RefreshRequest
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 */
data class RefreshResponse (

        @get:NotNull 
        @JsonProperty("accessToken") override val accessToken: String,

        @JsonProperty("clientToken") override val clientToken: String? = null
) : RefreshRequest{

}

